store/postgres/src/connection_pool.rs

```diff
@@ -331,12 +352,24 @@ impl ConnectionPool {
         // available.
         let timeout_seconds = if cfg!(test) { 30 } else { 6 * 60 * 60 };
         let pool = Pool::builder()
-            .error_handler(error_handler)
-            .event_handler(event_handler)
+            .error_handler(error_handler.clone())
+            .event_handler(event_handler.clone())
             .connection_timeout(Duration::from_secs(timeout_seconds))
             .max_size(pool_size)
```

Comment:
Should we also set `.min_idle(Some(1))` in the ordinary pool? This prevents the node from acquiring all of the connections already at startup, which might help with past issues we had when resetting nodes.

Reply:
Maybe - but that's the sort of subtle change that I'd prefer to do in isolation, since it can lead to a lot more blocking because connections might have to be (re)established more often. It's probably fine and not an issue, but it feels safer to do that separately.